<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_create_test">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>시험 만들기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load_test">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>시험 불러오기</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>230</y>
     <width>421</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>650</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>441</width>
     <height>651</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>시험1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>시험2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>시험3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>시험4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>시험5</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
